app/Listeners/Saml2Login.php

```diff
@@ -42,25 +42,20 @@ public function handle(Saml2SignedInEvent $event): void
             Auth::login($user);
             return;
         }
-
-        if (config('saml2.autoregister_new_users') === true) {
-            // Automatically register this new user.
-            $registerData = [
-                'email' => $email,
-                'fname' => strtok($email, '@'),
-                'lname' => '',
-                'institution' => '',
-                'password' => Hash::make(Str::random(40)),
-            ];
-            $registerController = new RegisterController();
-            $user = $registerController->create($registerData);
-            if ($user === null) {
-                Log::error("Error registering new SAML2 user: $email");
-                abort(500, 'Error registering new SAML2 user');
-            }
-            Auth::login($user);
-        } else {
-            abort(401);
```

Comment:
@zackgalbreath, Can you weigh in on this `abort`?  
It seems like signing in via SAML either should have always autoregister-ed or might reject people who haven't had accounts made separately but now want to sign in with SAML.

Reply:
Yes, this looks inconsistent with the "redirect to register" logic we have in OAuthController.

IIRC, the requested workflow was:
* CDash admin creates the user
* User logs in (via SAML2)

Reply:
Okay, thanks! I'll revert the changes to the SAML work since that's explicitly different.  We can talk to them next time to see if they want to maintain that functionality or not.